ffs/scheduler/scheduler.go

```diff
@@ -83,47 +83,48 @@ func New(ds datastore.TxnDatastore, l ffs.CidLogger, hs ffs.HotStorage, cs ffs.C
 	return sch, nil
 }
 
-// PushConfig queues the specified CidConfig to be executed as a new Job. It returns
+// PushConfig queues the specified StorageConfig to be executed as a new Job. It returns
 // the created JobID for further tracking of its state.
-func (s *Scheduler) PushConfig(iid ffs.APIID, cfg ffs.CidConfig) (ffs.JobID, error) {
-	return s.push(iid, cfg, cid.Undef)
+func (s *Scheduler) PushConfig(iid ffs.APIID, c cid.Cid, cfg ffs.StorageConfig) (ffs.JobID, error) {
```

Comment:
Same in here where we need to pass along the cid since it's not in `StorageConfig`